starknet_py/abi/v1/schemas.py

```diff
@@ -25,7 +25,7 @@ class FunctionBaseSchema(Schema):
     outputs = fields.List(
         fields.Nested(TypeSchema()), data_key="outputs", required=True
     )
-    state_mutability = fields.String(data_key="state_mutability", default=None)
+    state_mutability = fields.String(data_key="state_mutability", dump_default=None)
```

Comment:
I've changed it because of this warning: 
`RemovedInMarshmallow4Warning: The 'default' argument to fields is deprecated. Use 'dump_default' instead.
    state_mutability = fields.String(data_key="state_mutability", default=None)`
I will help us migrate in the future